### PR TITLE
use SciML instead

### PR DESCRIPTION
**Pull request by Datseris**

### CODE REVIEW

**Comment by Datseris** [comment]
@awage feel free to merge and tag here if tests pass, and then merge and tag https://github.com/JuliaDynamics/DynamicalSystems.jl/pull/164 . I'm at the airport now and wont come back here for quite some time!

File: src/period_return/transit_times_statistics.jl [hunk 1]
```diff
@@ -259,7 +261,7 @@ function exit_entry_times(ds::ContinuousDynamicalSystem, u0, εs, T;
     exits = [eT[] for _ in 1:length(εs)]
     entries = [eT[] for _ in 1:length(εs)]
 
-    # This callback fails, see https://github.com/SciML/DiffEqBase.jl/issues/580
+    # This callback fails, see https://github.com/SciML/SciMLBase.jl/issues/580
```
Comment (Datseris) [inline src/period_return/transit_times_statistics.jl:264]:
```suggestion
    # This callback fails, see https://github.com/SciML/DiffEqBase.jl/issues/580
```

**Comment by awage** [comment]
I didn't merge JuliaDynamics/DynamicalSystems.jl#164 since tests were failing. I prefer not to mess with version control.

**Comment by Datseris** [comment]
ok fair, i'll do it when i get the time. How did you tag a new release? Did you do it at the GeneralRegistry.jl as well?

**Pull request merged**